flush.sql

```diff
@@ -0,0 +1,15 @@
+BEGIN;
```

Comment:
Is this file used anywhere?

Reply:
Sorry, not sure how that made into the repo. No that's something I use personally.